roles/test-utils/mining-device/src/main.rs

```diff
@@ -276,6 +285,7 @@ impl Device {
             prev_hash: None,
             channel_id: None,
             sequence_numbers: Id::new(),
+            notify_changes_to_mining_thread: (true, notify_changes_to_mining_thread),
```

Comment:
could we move `(true, notify_changes_to_mining_thread)` to a type to improve readability? will also eliminate the `.0` \ `.1` ambiguity